glib/src/utils.rs

```diff
@@ -41,8 +41,6 @@ pub fn set_prgname(name: Option<&str>) {
 pub fn getenv<K: AsRef<OsStr>>(variable_name: K) -> Option<OsString> {
     #[cfg(not(windows))]
```

Comment:
Same as for the gdk-pixbuf changes. The `#[cfg]` need to go away so it compiles on Window

Reply:
okay. I am getting rid of these functions from the utils.rs and the Gir.toml files except the `mkstemp()` function